DEVELOPMENT.md

```diff
@@ -1,27 +1,20 @@
 # DEVELOPMENT
 
-[![CircleCI](https://circleci.com/gh/GuildCrafts/curriculum.svg?style=svg)](https://circleci.com/gh/GuildCrafts/curriculum)
-
 ## Dev Environment Setup
 
-
 Install and run Node v8.1.2
 
-
-#### Setup and run [mehserve](https://github.com/timecounts/mehserve)
```

Comment:
is [setup](https://www.google.com/search?q=define%3A+setup&oq=define%3A+setup&aqs=chrome..69i57j69i58.1694j0j1&sourceid=chrome&ie=UTF-8) a bad word?

Reply:
i dont mind the change. just curious